packages/tools/src/utilities/math/basic/BasicStatsCalculator.ts

```diff
@@ -0,0 +1,60 @@
+import { StatisticValue } from '../../../types/CalculatorTypes';
+import Calculator from '../../../types/CalculatorTypes';
+
+export default class BasicStatsCalculator extends Calculator {
+  max = -Infinity;
+  currentMax = 0;
+  sum = 0;
+  sumSquares = 0;
+  squaredDiffSum = 0;
+  count = 0;
+
+  /**
+   * This callback is used when we verify if the point is in the annotion drawn so we can get every point
+   * in the shape to calculate the statistics
+   * @param value of the point in the shape of the annotation
+   */
+  statsCallback = ({ value: newValue }): void => {
```

Comment:
I would call this `run` or something similar. And Also I would make the class an static class so that we don't do `new ...`

```
export default class BasicStatsCalculator extends Calculator {
  static max = -Infinity;
  static currentMax = 0;
  static sum = 0;
  static sumSquares = 0;
  static squaredDiffSum = 0;
  static count = 0;

  static statsCallback({ value: newValue }): void {
```